app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt

```diff
@@ -167,21 +167,30 @@ class RoomViewModel(
             localAudioTrack = it.createAudioTrack(
                 mapOf(
                     "user_id" to (localDisplayName ?: ""),
-                    "active" to true
+                    "active" to true,
+                    "type" to "audio"
                 )
             )
 
             var videoParameters = VideoParameters.presetHD169
             videoParameters = videoParameters.copy(
                 dimensions = videoParameters.dimensions,
-                simulcastConfig = videoSimulcastConfig.value
+                simulcastConfig = videoSimulcastConfig.value,
+                maxBitrate = TrackBandwidthLimit.SimulcastBandwidthLimit(
+                    mapOf(
+                        "l" to TrackBandwidthLimit.BandwidthLimit(150),
+                        "m" to TrackBandwidthLimit.BandwidthLimit(500),
+                        "h" to TrackBandwidthLimit.BandwidthLimit(1500)
+                    )
+                )
             )
 
             localVideoTrack = it.createVideoTrack(
                 videoParameters,
                 mapOf(
                     "user_id" to (localDisplayName ?: ""),
-                    "active" to true
+                    "active" to true,
+                    "type" to "camera"
```

Comment:
Does the screensharing need a special type too?

Reply:
screensharing has it already: https://github.com/membraneframework/membrane-webrtc-android/blob/master/app/src/main/java/com/dscout/membranevideoroomdemo/viewmodels/RoomViewModel.kt#L352